OpenSpeedTest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "SpeedTest by OpenSpeedTest": {
+    "containers": {
+      "speedtest": {
+        "image": "openspeedtest/latest",
+		"launch_order": 1,
+		"ports": {
+			"3000": {
+				"description": "SpeedTest http WebUI port. Suggested Default: 3000",
+				"host_default": 3000,
+				"label": "http WebUI port",
+				"ui": true
+			}
+		}
+      }
+    },
+    "description": "<p>SpeedTest by OpenSpeedTest™ is a free and open-source HTML5 network performance estimation tool. Written in Vanilla Javascript it only uses built-in Web APIs like <code>XMLHttpRequest</code> <code>(XHR)</code>, <code>HTML</code>, <code>CSS</code>, <code>JS</code>, &amp; <code>SVG</code>. No third-party frameworks or libraries are required.<p>Based on the official docker image: <a href='https://hub.docker.com/u/openspeedtest/latest' target='_blank'> https://hub.docker.com/u/openspeedtest/</a>, available for amd64 and arm64 architecture.</p>",
```

Comment:
@Hooverdan96 I think the code formatting here is a little much, maybe we just do bold:

Also I get a 404 on the link when clicked: https://hub.docker.com/u/openspeedtest/latest

Reply:
oops, yes, looks like I messed up when I went from the "top-level" docker link to the specific "latest" one, it needs to have `/r/` instead of `/u/` in the link. I'll fix that and remove the formatting for those thing entirely ... it really doesn't need to be that way (I had just copied -and corrected - the stuff from the docker page).